rollups/evm/connect-oracle/README.md

```diff
@@ -0,0 +1,65 @@
+##  Connect Oracle
+
```

Comment:
do we want to explain a bit here of what this guide is about, what does it do, what is the expected result, etc. ?
the rest lgtm

Reply:
added simple context